donkeycar/templates/cfg_complete.py

```diff
@@ -350,6 +350,10 @@
 LIDAR_LOWER_LIMIT = 90 # angles that will be recorded. Use this to block out obstructed areas on your car, or looking backwards. Note that for the RP A1M8 Lidar, "0" is in the direction of the motor
 LIDAR_UPPER_LIMIT = 270
 
+# TFMINI
+HAVE_TFMINI = True
```

Comment:
This should be `False` as the majority of users will not have that part installed.